var/spack/repos/builtin/packages/python/package.py

```diff
@@ -118,6 +118,7 @@ class Python(AutotoolsPackage):
     depends_on('tk', when='+tkinter')
     depends_on('tcl', when='+tkinter')
     depends_on('libuuid', when='+uuid')
+    depends_on('gettext')
```

Comment:
Is this for providing gettext support in python? https://docs.python.org/2/library/gettext.html

Reply:
When building `python@3.7.2%gcc@8.3.0`, the Python configure script found `libintl.h` (perhaps in the system paths somewhere?), but the corresponding `-lintl` inserted by configure failed because the responsible `gettext` installation wasn't in the library path. This was in the `_localemodule.c` file, but it looks like that code is indeed responsible for activating the `gettext` builtin module.

I'm actually unable to reproduce this failure now... so perhaps `gettext` should be an optional variant? Or should I just remove it for the time being?

Reply:
> I'm actually unable to reproduce this failure now... so perhaps gettext should be an optional variant? Or should I just remove it for the time being?

Based on the rest of your comment, `gettext` likely *should* be added as a dependency, since the python installation looks for it at some point (and uses a system version if it's available). The other option is to explicitly disable the python feature which requires `gettext` (it seems that most python installs have gotten by without it).

Reply:
There's a part of the `configure` script with the comment "check if we need libintl for locale functions" and tries to link against a `textdomain` function; there's no option to disable it.

Reply:
OK, it sounds like `gettext` should in fact be added as a dependency then.

Reply:
Sounds good. I also just verified that the header check for lintl is there in Python 2.7, so it seems like we shouldn't have to qualify the dependency with `when`.

Reply:
Can you move this higher up under pkgconfig? This section is for optional dependencies. It's a shame there's no way to disable this dependency.